ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-aro.yaml

```diff
@@ -65,15 +55,13 @@ tests:
   cron: 28 7 4 * *
   steps:
     cluster_profile: azure-qe
+    env:
+      ARO_CLUSTER_VERSION: 4.13.23
     test:
     - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_INTERMEDIATE411},${RELEASE_IMAGE_INTERMEDIATE412},${RELEASE_IMAGE_INTERMEDIATE413},${RELEASE_IMAGE_TARGET}"
-        > ${SHARED_DIR}/upgrade-edge
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE413},${RELEASE_IMAGE_TARGET}" >
```

Comment:
The same comment as https://github.com/openshift/release/pull/50980/files#r1569815798, if this upgrade is only one-hop uprade, we do not need the `set-upgrade-releases` step here.

Reply:
4.14 one should be fixed no.